src/scripts/main.js

```diff
@@ -1 +1,19 @@
 'use strict';
+
+const ul = document.querySelector('.gallery__list');
+const largeImg = document.querySelector('.gallery__large-img');
+
+ul.addEventListener('click', (e) => {
+  const clickEvent = e;
+
+  clickEvent.preventDefault();
+
+  if (!clickEvent) {
+    // eslint-disable-next-line no-useless-return
+    return;
+  }
+
+  const link = e.target.closest('.list-item__link');
+
+  largeImg.src = link.href;
```

Comment:
There is a potential issue here if `link` is null. You should check if `link` is not null before accessing `link.href` to avoid runtime errors. Consider adding a condition to ensure `link` is a valid element.